src/main/java/com/alkemy/ong/domain/usecase/impl/CommentServiceImpl.java

```diff
@@ -1,12 +1,20 @@
 package com.alkemy.ong.domain.usecase.impl;
 
+import com.alkemy.ong.domain.model.Comment;
 import com.alkemy.ong.domain.repository.CommentRepository;
 import com.alkemy.ong.domain.usecase.CommentService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @RequiredArgsConstructor
 public class CommentServiceImpl implements CommentService {
     private final CommentRepository commentJpaRepository;
+
+    @Override
+    @Transactional
+    public Long createComment(Comment comment) {
+        return commentJpaRepository.save(comment).getId();
```

Comment:
Buscar la entity de user y new en sus repositorios. Si no existen devolver not found. Por ejemplo para user:

```java
        User user = userJpaRepository.findById(comment.getUser().getId())
                .orElseThrow(() -> new NotFoundException(comment.getUser().getId()));
        comment.setUser(user);
```